portality/constants.py

```diff
@@ -0,0 +1,19 @@
+APPLICATION_STATUS_ACCEPTED = "accepted"
```

Comment:
Should we be putting user roles in this file too?

Reply:
In the long term, I think yes.  I didn't bother for this PR, I only put in stuff that I touched.